src/_guides/language/coming-from/js-to-dart.md

```diff
@@ -2555,26 +2706,25 @@ void main() {
 }
 ```
 
+
+
 ### Public and private members
 
 Like JavaScript, Dart has no access modifier keywords:
-all class members are public by default. 
+all class members are public by default.
 
-While private class members are not yet officially
-part of JavaScript because they are not part of any
-published EcmaScript standard,
-a proposal for this has been completed and is ready
-to be included in the next publication of the standard.
+JavaScript will include private class members in the next
+practical revision of the EcmaScript standard.
```

Comment:
This whole talk about the EcmaScript spec for private members not being complete is confusing and I'm not sure it should be included. Either way, it looks like PrivateIdentifiers were part of EcmaScript 2022 which has been released.

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields

Reply:
I checked the reference specifically; privates are in stage 4, which means they will be in the next practical revision. That's why I phrased it as I did.